pyproject.toml

```diff
@@ -136,7 +136,7 @@ exclude = [
   "^src/instructlab/train/lora_mlx/models/mixtral\\.py$",
   "^src/instructlab/train/lora_mlx/models/models\\.py$",
   "^src/instructlab/utils\\.py$",
-  "^src/instructlab/model/backends/llama\\.py$",
+  "^src/instructlab/model/backends/llama_cpp\\.py$",
```

Comment:
Are there still typing violations in the file?

Reply:
Yes it does, this file used to be `server.py`. That PR should fix it https://github.com/instructlab/instructlab/pull/1275. Thanks!